cmd/cluster-agent/admissioncontroller/server.go

```diff
@@ -0,0 +1,69 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// Package admissioncontroller implements the agent admission webhook
+// endpoint. This server receives k8s objects from the cluster and
+// preforms changes or admission control to such objects.
+//
+// More info: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/
+package admissioncontroller
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"net"
+	"net/http"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+var (
+	server *http.Server
+)
+
+func getListener(port int) (net.Listener, error) {
```

Comment:
I think I left this comment before, but this is also not used elsewhere, let's remove the function and simply inline the listener creation.

Reply:
You resolved this comment but didn't remove this function, which is now unused.